builtin/game/chat.lua

```diff
@@ -914,14 +931,39 @@ core.register_chatcommand("time", {
 			core.log("action", name .. " sets time to " .. new_time)
 			return true, "Time of day changed."
 		end
+		-- HH:mm notation with relative set time support
+		hour = string.trim(hour)
+		minute = string.trim(minute)
```

Comment:
What do you need to trim? The expression above does not match spaces.
These lines were added in 70cac9d5 and kept them here for clarification.